### ISSUE TITLE
pretrained model mode (ADV/TB) to skip training

### ISSUE THREAD

**Issue by Dimitri1** [facebookresearch/dlrm#52]
Hello,

First, thank for the code quality (on pytorch at least).
My problem is simple.
I need to perform accuracy validation of dlrm (in ACV and TB challenge) with my company DSP target. So, I'm not interested on the training phase. Unless I misunderstood the code, it is not possible to load a pretrained model to only preform the inference.
Unless there is another solution, it should be sexy to support a mode like "pretrained model inference".

PS: By the way, I think that args.inference_only is ambiguous.

**Comment by Dimitri1** [comment]
My first post is incorrect, such mod is done with  --load-model="path to model.pt" --inference-only, so my problem now becomes : Where can we find pretrained ADV and TB dlrm model.

**Comment by mnaumovfb** [comment]
At the moment, we do not provide pre-trained models for Criteo Kaggle Display Advertising Challenge and Terabyte datasets. We have considered this in the past https://github.com/facebookresearch/dlrm/issues/40, so I'll take this as another request for it.

You should be able to train a model from scratch in a few hours when interfacing with Criteo Kaggle Display Advertising Challenge dataset. However, the Terabyte dataset would take significantly longer to train.

**Comment by Dimitri1** [comment]
mnaumovfb, thanks for this confirmation.
I'll be able to train ADV myself but it will be much more difficult 
for me to do the same with TB. I just finish day_0 downloading and it takes
48 GB after gzip, 2 hour to load splits with data API, and finally I don't have descent GPU
for training. It would be very nice to provide a trained .pt for TB ? Maybe  I'm quite marginal when I say that I'm only interested in inference accuracy, but I don't understand the reason why 
such pretrained models are not provided, especially when it requires such efforts.

**Comment by mnaumovfb** [comment]
We will work on providing pre-trained models. Give us a bit of time for it. 

I'll let you know when they become available.

**Comment by Dimitri1** [comment]
Super  mnaumovfb, thank you.
Should I close the issue ?

**Comment by mnaumovfb** [comment]
Let's keep it open for now. We will close once the pre-trained models become available.

**Comment by mnaumovfb** [comment]
I have added a link to pre-trained models to the README. They are available under [CC-BY-NC license](https://creativecommons.org/licenses/by-nc/2.0/).

**Comment by Dimitri1** [comment]
Thank you mnaumovfb.
